mscore/excerptsdialog.cpp

```diff
@@ -162,12 +165,35 @@ void MuseScore::startExcerptsDialog()
     if (cs == 0) {
         return;
     }
-    ExcerptsDialog ed(cs->masterScore(), 0);
+    MasterScore* ms = cs->masterScore();
+    if (cv->drawingScore()->movements()->size() > 1) {
+        Q_ASSERT(cv->drawingScore() == Album::activeAlbum->getCombinedScore());
+        if (!Album::activeAlbum->checkPartCompatibility()) {
+            std::cout << "Parts not matching..." << std::endl;
+            QMessageBox msgBox;
+            msgBox.setWindowTitle(QObject::tr("Incompatible parts :-("));
```

Comment:
I like a bit of humour as much as the next person, but could we remove the `:-(` from the window title, please? :)

Reply:
We can have a frownie icon in the msgBox maybe

Reply:
maybe if its hidden 🙃